styles/namemc/catppuccin.user.css

```diff
@@ -358,17 +358,12 @@
       }
     }
 
+    /* "recent names" */
     table {
-      tbody tr {
-        td.text-nowrap.text-ellipsis a {
+      tbody tr.border-top {
+        td a {
           color: @accent-color !important;
         }
-
-        &.border-top {
-          td a {
-            color: @accent-color !important;
-          }
-        }
       }
```

Comment:
I don't think any of this section is needed. Try removing it on your end, I'm not seeing any visible changes. Looks like maybe that link is already themed?

Reply:
yeah it looks like what's happening here is _likely_ the same as the previous one; overriding the variable that does the exact same thing :P